honeybee_vtk/model.py

```diff
@@ -54,7 +58,8 @@ class Model(object):
     """
     def __init__(
         self, model: HBModel,
-        load_grids: SensorGridOptions = SensorGridOptions.Ignore
+        load_grids: SensorGridOptions = SensorGridOptions.Ignore,
+        load_cameras: bool = True
```

Comment:
I have a hard time to communicate this one! I think we should fully remove this option and ALWAYS load the cameras since they are small objects. That's why I think we probably don't need an option like this.

```suggestion
```

Reply:
This is clear now. Complied.